lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -562,12 +562,15 @@ function dosomething_signup_get_mbp_params($account, $node, $opt_in) {
   $url_options = array(
     'language' => $language,
     'absolute' => TRUE,
-    'prefix' => strtolower(dosomething_global_convert_language_to_country($language)) . '/',
     'query' => array(
       'source' => 'node/' . $node->nid,
     ),
   );
 
+  if ($country = strtolower(dosomething_global_convert_language_to_country($language))) {
```

Comment:
Interesting that both @sergii-tkachenko and I misread this. @blisteringherb that's a sign to me that it's idiomatic PHP that we should remove in favor of something more explicit. If you really want to keep the construct, please add a comment explaining when and how it evaluates to FALSE, for Future Generations.

(Also a sign that @sergii-tkachenko and I need to read more carefully.)